dcos/package.py

```diff
@@ -1048,19 +1048,20 @@ def check_version(self, min_version, max_version):
         :returns: None
         """
 
-        version = LooseVersion(self.get_version())
-        if not (version >= min_version and version < max_version):
+        package_version = LooseVersion(self.get_package_version())
+        if not (package_version >= min_version and
+                package_version < max_version):
             raise DCOSException((
                 'Unable to update source [{}] because version {} is '
                 'not supported. Supported versions are between {} and '
                 '{}. Please update your DCOS CLI.').format(
                     self._source.url,
-                    version,
+                    package_version,
                     min_version,
                     max_version))
 
-    def get_version(self):
-        """Returns the version of this registry.
+    def get_package_version(self):
+        """Returns the package version of this registry.
```

Comment:
This is the version of the registry, not a package.  So it was correct before.  Can you go through again and make sure all of these version renamings are correct?